network/alsp/params.go

```diff
@@ -0,0 +1,30 @@
+package alsp
+
+// To give a summary with the default value:
+//  1. The penalty of each misbehavior is 0.01 * misbehaviorDisallowListingThreshold = -86.4
+//  2. The penalty of each misbehavior is decayed by a decay value at each decay interval. The default decay value is 1000.
+//     This means that by default if a node misbehaves 1000 times in a second, it gets disallow-listed, and takes 86.4 seconds to recover.
```

Comment:
```suggestion
//     This means that by default if a node misbehaves 100 times in a second, it gets disallow-listed, and takes 86.4 seconds to recover.
```

Default penalty is 1% of threshold, so 100 misbehaviours within 1 second (before a decay) results in block-listing.